Random_numbers/include/CGAL/Random.h

```diff
@@ -228,21 +228,29 @@ class Random {
     boost::rand48 rng;
 };
 
-#ifndef CGAL_HEADER_ONLY
-// Global variables
-// ================
-CGAL_EXPORT extern Random default_random;
-#endif // CGAL_HEADER_ONLY
-
 #ifdef CGAL_HEADER_ONLY
+
 inline Random& get_default_random()
 {
   static Random default_random;
   return default_random;
 }
+
+#ifndef CGAL_NO_DEPRECATED_CODE
+  namespace { CGAL_DEPRECATED CGAL::Random& default_random = get_default_random(); }
+#endif // CGAL_NO_DEPRECATED_CODE
+
 #else // CGAL_HEADER_ONLY
+
+// Global variables
+// ================
+#ifndef CGAL_NO_DEPRECATED_CODE
+  CGAL_EXPORT extern Random default_random;
```

Comment:
@lrineau @cjamin Here (in non header only mode) we cannot mark the global variable deprecated because we use this variable in the get_default_random.

A possibility could be to mark here this variable deprecated, and to remove the deprecated warning line 252 before the line " return default_random;" but I don't know it this is possible, and if yes I don't know how to do this...